pkgs/by-name/rg/rgl/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  nix-update-script,
+  versionCheckHook,
+}:
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```
use `finalAttrs` instead